src/simple_tracker_detection_provider/simple_tracker_detection_provider/bgs_detector_node.py

```diff
@@ -28,9 +28,9 @@ def __init__(self, subscriber_qos_profile: QoSProfile, publisher_qos_profile: Qo
 
     # setup services, publishers and subscribers
     self.sub_masked_background_frame = self.create_subscription(Frame, 'sky360/frames/masked_background/v1', 
-      self.masked_background_frame_callback, subscriber_qos_profile)
-    self.pub_key_points = self.create_publisher(KeyPointArray, 'sky360/detector/bgs/key_points/v1', publisher_qos_profile)
-    self.pub_bounding_boxes = self.create_publisher(BoundingBoxArray, 'sky360/detector/bgs/bounding_boxes/v1', publisher_qos_profile)   
+      self.masked_background_frame_callback, 10)#, subscriber_qos_profile)
+    self.pub_key_points = self.create_publisher(KeyPointArray, 'sky360/detector/bgs/key_points/v1', 10)#, publisher_qos_profile)
```

Comment:
Should there be a named constant for the integer?

Reply:
Well spotted, so this is a shortcut way of defining some sort of QOS i.e. queue length is 10 messages in length. This simple QOS is temporary as I want to do proper QOS job (as I had in place prior to refactoring message types), so at the moment I just don't want to over complicate while I standardise message types. As soon as I am done with standardising message types I will provide an implementation for QOS.